### PR TITLE
Add "T-Rex 🦖" sticker pack

### PR DESCRIPTION
**Pull request by JanOstrowka**
https://signalstickers.com/pack/05480210c5eec8191c2dde3d248364c7?key=ea1f7d6f3b94eab53cb3cc472fbeb78b502fc8e15082add959a6cfca7150e1b2

### CODE REVIEW

**Comment by romainricard** [comment]
Thank you for all your contributions @JanOstrowka !

File: stickers.yml [hunk 1]
```diff
@@ -19829,3 +19829,22 @@ e374df1d3ddbc818ee7f60972157c034:
...
+    - Dino
+    - Apple
+    - Memoji
+    - Emoji
+    - Iphone
+    - IPad
+    - Green
+    - Animal
+  nsfw: false
+  original: false
```
Comment (romainricard) [inline stickers.yml:19850]: Isn't this one also original?

Comment (JanOstrowka) [reply stickers.yml:19850]: It is the stock Memoji version, imported from Apple keyboard, in iMessage. I converted the "artworks", but did not design them myself. Also they are available to any Apple user, who has access to Memoji on their device.

image

btw ~ I am planning to add the rest of them, too

Comment (romainricard) [reply stickers.yml:19850]: Ho, great, I don't use iMessage so I had no idea. Thanks!

**Pull request merged**